test/perl5/libdnf5/rpm/test_package_query.t

```diff
@@ -34,6 +34,10 @@ my $tmpdir = tempdir("libdnf5_perl5_unittest.XXXX", TMPDIR => 1, CLEANUP => 1);
 $base->get_config()->get_installroot_option()->set($libdnf5::conf::Option::Priority_RUNTIME, $tmpdir."/installroot");
 $base->get_config()->get_cachedir_option()->set($libdnf5::conf::Option::Priority_RUNTIME, $tmpdir."/cache");
 
+# Prevent loading plugins from host
+$base->get_config()->get_plugins_option()->set("False");
```

Comment:
Use <code>False</code> without quotes. As a string it evaluates to True.

Reply:
This is funny - set() accepts a string and then it converts it to bool. `OptionBool` config class does that and there is wide range of definition what false means.

Reply:
That's Perl. I thought it's Python. In Perl <code>"false"</code> means true, <code>false</code> is a Perl glob identifier (so called a bare word). In Perl, using <code>0</code> is the best option.

Reply:
I see, OptionBool::get_default_false_values() recognizes "false" string.